content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md

```diff
@@ -166,6 +166,42 @@ INTO "example-db"."example-rp"."example-measurement"
 {{% /code-tab-content %}}
 {{< /code-tabs-wrapper >}}
 
+##### Write pivoted data to InfluxDB
+InfluxDB 1.x pivots queried fields into columns and returns tables with a column for each field.
```

Comment:
Is there a definition of "pivot" we could link to or mention here, or is it clear in the current context? Or is this sentence itself the definition?

Reply:
Good call. Updated this.